src/Network/Network/Network.csproj

```diff
@@ -41,10 +42,6 @@
     <Folder Include="RouteServerPeer\" />
   </ItemGroup>
   
-  <ItemGroup>
-    <ProjectReference Include="..\Network.Management.Sdk\Network.Management.Sdk.csproj" />
-  </ItemGroup>
-
```

Comment:
Please revert the change to remove the local built "Network.Management.Sdk". And remove the nupkg files under tools/LocalFeed.

Reply:
Hi @YanaXu ,we have kind of dependency on updated SDK related PR ([https://github.com/Azure/azure-rest-api-specs/pull/24965](https://github.com/Azure/azure-rest-api-specs/pull/24965/files)),
Once that PR get merge then we will update it here.
If you have any suggestion to resolve this dependency, please suggest us.
Thank you.